lib/Synchronise/Document.hs

```diff
@@ -41,13 +43,14 @@ data Document = Document
 
 makeLenses ''Document
 
-instance ToJSON Document where
-    toJSON = _documentContent
+$(deriveJSON defaultOptions ''Document)
```

Comment:
This is probably wrong. What are we going to do with the JSON of a `Document` (as opposed to the `Value` which is its `_documentContent`)?